vcluster/_fragments/eso.mdx

```diff
@@ -12,24 +10,38 @@ import ProAdmonition from '../_partials/admonitions/pro-admonition.mdx'
 
 # External secrets integration
 
+<Admonition type="warning" title="Deprecation Notice">
+  The sync config for this integration has been changed.
+  Configuration for each resource type was moved under their respective `toHost` and `fromHost` keys (see [reference](#config-reference) below).
+
+  Label selectors are now available for all resources and follow the from used by the upstream Kubernetes API.
+
+  The old config keys are deprecated starting with 0.27.0 and is **removed in 0.29.0**.
+</Admonition>
+
 To enable the external secret integration, set the following fields:
 
 ```yaml
 integrations:
   externalSecrets:
     enabled: true
     sync:
-      externalSecrets:
-        enabled: true
-      stores:
-        enabled: true
-      clusterStores:
-        enabled: true
+      toHost:
+        externalSecrets: {}
```

Comment:
What's the point of this line ? I thought it was always default enabled. If it's just the selectors, I would remove this.

Reply:
I thought it may be confusing if one field is missing.
You're right it's not necessary.

Technically, it should be completely fine to only enable the integration and no additional resources.
Do we want to document a more complete example or the minimum?

Reply:
We aim to have a minimum implementation at the top so people can get started quickly. Then the bottom will have full configuration options.